scripts/generate_workers_cfg.sh

```diff
@@ -90,7 +90,12 @@ generate_workers () {
             fi
 
             add_section volumes
-            add_item \${SHARED_WORK_DIRECTORY}:/data
+            #add_item \${SHARED_WORK_DIRECTORY}:/data
```

Comment:
it's not deprecated here, but removed support ...

Reply:
Compliance is now take into account.